week-3/slideshow/index.html

```diff
@@ -1,14 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
-    <script defer src="slideshow.js"></script>
-  </head>
-  <body>
-    <img id="carousel-img" src="./assets/cute-cat-a.png" alt="cat-pic" />
-    <button type="button" id="backward-btn">Backwards</button>
-    <button type="button" id="forward-btn">Forward</button>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/week-3/slideshow/style.css">
+  <title>Slideshow</title>
```

Comment:
![image](https://github.com/CodeYourFuture/Module-JS2/assets/118762745/699250b7-fcfe-4d8a-9889-620503c1bb26)
Be very careful when readinginstructions! The title should have been image carousel!